tensorflow_recommenders_addons/dynamic_embedding/core/kernels/lookup_impl/lookup_table_op_gpu.h

```diff
@@ -49,6 +49,84 @@ struct ValueArray : public ValueArrayBase<V> {
 template <class T>
 using ValueType = ValueArrayBase<T>;
 
+template <typename T>
+class HostFileBuffer {
```

Comment:
It looks like repeated code with `HostFileBuffer` in lookup_table_op_cpu.h, if yes, recommending you move them to tensorflow_recommenders_addons/dynamic_embedding/core/utils/host_file_buffer.h

Reply:
Accept